libs/utils/src/cast_vote_records.ts

```diff
@@ -32,6 +38,57 @@ export function getCurrentSnapshot(cvr: CVR.CVR): Optional<CVR.CVRSnapshot> {
   );
 }
 
+/**
+ * Because there is no place for "Absentee" vs. "Precinct" ballot on the CDF
+ * for cast vote records, we shove it into the `OtherStatus` field of the
+ * `CVRSnapshot`. This tool formats that metadata, which can be spread into
+ * a `CVRSnapshot`.
```

Comment:
I meant, like, you can use the spread operator to add the multiple necessary attributes into a `CVRSnapshot` object. Can add a usage example instead of trying to explain it.